### PR TITLE
Add pending_version state handling, Fix getGist async issue

### PR DESCRIPTION
**Pull request by k-lpmg**
### 1.  auto-indent to sources
- While working, vscode automatically indented.

### 2. fix fetch_app_state
- Add in_review_version_info, pending_version_info, latest_version_info to `fetch_app_status.rb`
- I've been testing it for a month and it's been working fine without any issues.
- [Issue Link](https://github.com/techinpark/appstore-status-bot/pull/10#issuecomment-1023798508)

### 3. getGist async issue
- If `getGist` is executed asynchronously and the response is slower than execution of `exec fetch_app_status`, there is a case where the contents of the gist cannot be loaded and the duplicate app state is sent to Slack.
- So, I created the main function and applied await to getGist.
``` js
const main = async () => {
  await getGist();

  exec(
    "ruby Sources/fetch_app_status.rb",
    { env: env },
    function (err, stdout, stderr) {
      if (stdout) {
        var apps = JSON.parse(stdout);
        console.log(apps);
        for (let app of apps) {
          checkVersion(app);
        }
      } else {
        console.log("There was a problem fetching the status of the app!");
        console.log(stderr);
      }
    }
  );
};

...

main()
```

## Third party
``` diff
+ "request-promise-native": "^1.0.9"
```

## PR Links
- https://github.com/k-lpmg/appstore-status-bot/pull/1
- https://github.com/k-lpmg/appstore-status-bot/pull/2
- https://github.com/k-lpmg/appstore-status-bot/pull/3

### CODE REVIEW

**Comment by techinpark** [comment]
@k-lpmg Thanks for contribution again!  I will check this feature soon. 👍

File: Sources/fetch_app_status.rb [hunk 1]
```diff
@@ -4,28 +4,32 @@
 
 def get_app_state(app)
   
-  version_info = app.get_live_app_store_version
   edit_version_info = app.get_edit_app_store_version
-  appstore_version_info = app.get_app_store_versions.first
+  in_review_version_info = app.get_in_review_app_store_version
+  pending_version_info = app.get_pending_release_app_store_version
+  latest_version_info = app.get_latest_app_store_version
```
Comment (techinpark) [inline Sources/fetch_app_status.rb:10]: I didn't know that. `get_in_review_app_store_version`  Thank you!

**Review by techinpark — APPROVED** [review]
Great 💯   Thanks for contributions @k-lpmg !

File: Sources/fetch_app_status.rb (hunk 1 above)
Comment (k-lpmg) [reply Sources/fetch_app_status.rb:10]: `get_in_review_app_store_version` returns the status when the app store reviewer has started the build review process.

**Pull request merged**